### DISCUSSION TITLE
We are having multiple issues

### DISCUSSION THREAD

**Discussion by SriharshaWRGrace** [microsoft-search/pnp-modern-search#1971] [Q&A]
We are currently using PnP Modern Search v4 to showcase the videos across SharePoint and OneDrive

There are couple of issues we are facing:

1. My OneDrive recordings doesn't show up? I can see random OneDrive videos but not mine. Any configuration needed?
2. Is there a way to have an in place(pop out) video player as opposed to going to the page where they are hosted?
3. Even though we have 4 filters configured only 2 filters are displayed at a time? Is there a limitation?

**Comment by FranckyC** [comment]
Hi @SriharshaWRGrace. 

> My OneDrive recordings doesn't show up? I can see random OneDrive videos but not mine. Any configuration needed?

Can you share you query configuration and settings?

> Is there a way to have an in place(pop out) video player as opposed to going to the page where they are hosted?

You could try to use the `<pnp-filepreview>` component with the URL of your video (https://microsoft-search.github.io/pnp-modern-search/extensibility/web_components_list/#ltpnp-filepreviewgt)

> Even though we have 4 filters configured only 2 filters are displayed at a time? Is there a limitation?

Can you share the configuration details? If you've setup refiners, they only appear if your content actually contain values for these properties. If you don't see them it is probably because properties are empty for your search.

**Reply by SriharshaWRGrace** [reply]
Hi @FranckyC .

> Can you share you query configuration and settings?

We are using the "LocalVideoResults" from SharePoint Search. Please let us know if that helps


> You could try to use the <pnp-filepreview> component with the URL of your video (https://microsoft-search.github.io/pnp-modern-search/extensibility/web_components_list/#ltpnp-filepreviewgt).

We tried the above however the interface have some issues. When we click on a video the preview show up. However when we click on another video then the new video show up with audio from both current and the previous video. 

> Can you share the configuration details? If you've setup refiners, they only appear if your content actually contain values for these properties. If you don't see them it is probably because properties are empty for your search.

For all the above is there a documentation on how to configure the search and the filters? We see that the documentation on github show how to configure the webparts but how do we configure the tenant level settings so that it show all the results from SharePoint and Onedrive? And what are the different configurations needed? Please help.
